app/graphql/schema.graphql

```diff
@@ -102,6 +102,7 @@ type Profile implements Node & RulesPreload {
   majorOsVersion: String!
   name: String!
   policy: Profile
+  policyType: String!
```

Comment:
Food for thought: If we expose it as `type` in GQL, consumer can request it with:
```
profile(id: $id) {
  policy {
    type
  }
}
```

That gets us closer to v2 and could ease the migration.

Reply:
I could see us implementing it that way, but today, the `policy` in graphql is `null` for profiles without a policy, even though they technically also have a policy type. Having this top level attribute adds parity with the REST API's profile resource and helps handle external reports without any additional logic in the frontend.

As an aside: V2 is in regards to a new REST API, not graphql. IMO we can expose real policy IDs and other policy attributes if it makes sense here before we do the same in a V2 REST API. The most ideal case IMO would be to switch the UI to use the REST API and move away from using GraphQL entirely, though we haven't started using REST in any of our frontend components.